### ISSUE TITLE
Increased KCO

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6497]
**Preferred term label:** 
Increased KCO

**Synonyms**

**Definition (free text, please give PubMed ID)**
Increased diffusion capacity of the lung for carbon monoxide (CO) rate constant (efficiency index); PMID: 11334115

**Parent term (use hpo.jax.org/app)**
Anormal KCO

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Genetic neuromuscular disease ORPHA:183497
Idiopathic pulmonary haemosiderosis ORPHA:99931
Diffuse alveolar haemorrhage ORPHA:90060
**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
